message.d.ts

```diff
@@ -353,6 +353,11 @@ export namespace MessageEntity {
     /** For “pre” only, the programming language of the entity text */
     language?: string;
   }
+  export interface CustomEmojiMessageEntity extends AbstractMessageEntity {
+    type: "custom_emoji";
+    /** For “custom_emoji” only, unique identifier of the custom emoji. Use getCustomEmojiStickers to get full information about the sticker */
+    custom_emoji_id?: string;
```

Comment:
Are you sure this is optional?

Reply:
Ah, because it has its own interface, it become not optional, right?
Sorry my bad, I finally realized that

Reply:
No. What I am trying to ask is, are you sure that updates containing `custom_emoji` message entities may not have `custom_emoji_id` property?

For example,
- `pre` entities have their `language` property optional because sometimes updates will not have `language` property in the `pre` entity.
- But `url` entities **will** have `url` property.

Reply:
I think they should have

because `custom_emoji_id` is unique identifier of the custom emoji
So if the type of the message entity is `custom_emoji`, then there should be an `custom_emoji_id`

Reply:
What do you think?

Reply:
Yes, I think so. You could do that, or wait for the review of the main maintainer of this repo (@\KnorpelSenf), and do as he suggest.

Reply:
Thank you very much